.prettierrc.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import("prettier").Config} */
+module.exports = {
```

Comment:
Do we want this file to be **CommonJS** or keep the **ESM** from before? https://prettier.io/docs/en/configuration.html#basic-configuration

Reply:
Changed to ESM.